topics/assembly/tutorials/flye-assembly/tutorial.md

```diff
@@ -0,0 +1,253 @@
+
+## Get data from Zenodo
+
+> ### {% icon hands_on %} Hands-on: Data upload from Zenodo
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo](https://zenodo.org/record/5702408)
+>
+>    ```
+>   https://zenodo.org/record/5702408/files/SRR8534473_subreads.fastq.gz?download=1
+>   https://zenodo.org/record/5702408/files/SRR8534474_subreads.fastq.gz?download=1
+>   https://zenodo.org/record/5702408/files/SRR8534475_subreads.fastq.gz?download=1
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>
+> 3. Rename the datasets
+> 4. Check that the datatype are  `fastqsanger.gz`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+>
+{: .hands_on}
+
+## Get data from JGI website
+
+> ### {% icon hands_on %} Hands-on: Data upload from JGI website
+>
+> 1. Create a JGI account in registration page of JGI : [JGI registration](https://contacts.jgi.doe.gov/registration/new)
+> 2. Sign in JGI Genome Portal [JGI Genome Portal](https://genome.jgi.doe.gov/portal/)
+> 3. Genome assembly is available here : [JGI Mucor mucedo](https://genome.jgi.doe.gov/portal/Mucmuc1/Mucmuc1.download.html)
+> 4. Import fasta assembly file `Mucmuc1_AssemblyScaffolds.fasta` on your computer locally
+> 5. Upload this file on Galaxy
+>
+> 2. Check that the datatype is `fasta`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+>
+{: .hands_on}
+
+# Genome Assembly
+
+## Assembly with **Flye**
+
+We will use *Flye*, a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PacBio / ONT reads as input and outputs polished contigs. Flye also has a special mode for metagenome assembly. All informations about Flye assembler are here : [Flye](https://github.com/fenderglass/Flye/)
+
+> ### {% icon hands_on %} Hands-on: Assembly
+>
+> 1. {% tool [Flye](toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye/2.9+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input reads"*: the three sequencing datasets
+>    - *"Mode"*: `PacBio raw`
+>    - *"Number of polishing iterations"*: `1`
+>    - *"Reduced contig assembly coverage"*: `Disable reduced coverage for initial disjointing assembly`
+>    
+>     The tool produces four datasets : consensus, assembly graph, graphical fragment assembly and assembly info
+
+
+{: .hands_on}
+
+> ### {% icon question %} Question
+>
+> 1. What the different datasets obtained correspond to ?
+
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. The first dataset (consensus) is a fasta file containing the final assembly (1461 contigs).
```

Comment:
I got 1482, I guess the result is not always the same, you should warn about that

Reply:
done